ogc-common/src/main/java/io/arlas/server/ogc/common/utils/XmlUtils.java

```diff
@@ -157,8 +164,22 @@ private static void writeElementForType(XMLStreamWriter writer, String nameToDis
         }
     }
 
-    private static void writeElement(XMLStreamWriter xmlStream, String nameToDisplay, String value, String uri, String prefix) throws XMLStreamException {
+    private static void writeElement(XMLStreamWriter xmlStream, String nameToDisplay, String value, String uri, String prefix) throws XMLStreamException, ArlasException {
         nameToDisplay = replacePointPath(nameToDisplay);
+        if (nameToDisplay != null && nameToDisplay.length() > 0) {
```

Comment:
If you have apache lib, there is this StringUtils.isEmpty() or isBlank() (also trims) ;-)

Reply:
Fixed